src/components/TargetComponents/TargetDetail.tsx

```diff
@@ -32,6 +33,7 @@ function stemSection(stem: IStem[]): ContentSection[] {
 
   return sections;
 }
+
 // tslint:disable: no-non-null-assertion
 function handleFractions(taskModels: ITaskModel[]) {
```

Comment:
I was going to resume development on this but I'm not sure about the scope of the task and there isn't a task for this in Jira. Should we remove this method or does it have to be included elsewhere?

Reply:
I agree, this is really a parser and belongs in the main content parse util file. So I moved it there and added it to the parser list.

Reply:
Since you replaced this function, it needs to be removed.